shared/modules/module.go

```diff
@@ -18,9 +19,3 @@ type InterruptableModule interface {
 	Start() error
 	Stop() error
 }
-
-type InitializableModule interface {
```

Comment:
When we talk about `code clarity` ; here's another example where we loosen interfaces. It's difficult for me to say that the `V1 - Structs` design is definitely `clearer`